### DISCUSSION TITLE
插件部署在自己的服务器，自动更新机制（windows）

### DISCUSSION THREAD

**Discussion by wfk007** [ronggang/PT-Plugin-Plus#1150] [Q&A] [answered]
需求：我开发了一个 manifest v3 的 Chrome 插件，希望部署在自己的服务器上，windows 用户安装后可以自动更新

现状：看了 Chrome 的文档 https://developer.chrome.com/docs/extensions/mv3/linux_hosting/ 发现只有 Linux 是唯一支持可以不用部署在 Chrome web store 的操作系统

![image](https://user-images.githubusercontent.com/22289445/198271265-56ce0fdf-cc1f-44ad-996a-50b428f9e256.png)

我也尝试过基于 xml 的 update_url 的方案，把 updates.xml 放在 cdn 上，在 xml 中配置最新包的 crx 链接和 version 信息，使用注册表修改 ExtensionInstallAllowlist，修改注册表，配置 updates.xml，使用注册表安装 crx
尝试过上面的方案，也查阅了很多文档，还是无法自动更新

想请教一下大佬们，如何才能在 windows 上实现插件的自动更新机制！

**Comment by zgpinguo** [comment]
找到最终方案了不

**Reply by ted423** [reply]
现在这个扩展已经可以自动更新了吧

**Comment by Rhilip** [comment, marked as answer]
问题和本插件无关，不在解答范围。
请自己参照插件的相关实现。
